src/utils/urls.js

```diff
@@ -1,5 +1,12 @@
+function check() {
+  if (process.env.REACT_APP_LOCAL_ENV === 'true') {
```

Comment:
This condition needs to be changed, check is process is present first

Reply:
@fragm3 it works fine currently, if i build without setting the `REACT_APP_LOCAL_ENV` environment variable, it builds the normal version of the page.

Reply:
what do you suggest ?

Reply:
`process && process.env.REACT_APP_LOCAL_ENV === 'true'`